packages/node_modules/@webex/plugin-meetings/src/reconnection-manager/index.js

```diff
@@ -280,7 +297,11 @@ export default class ReconnectionManager {
         Metrics.postEvent(reconnectMetric);
         if (reconnectError instanceof NeedsRejoinError) {
           // send call aborded event with catogery as expected as we are trying to rejoin
-          return this.rejoinMeeting(reconnectError.wasSharing);
+
+          if (this.autoRejoinEnabled) {
+            return this.rejoinMeeting(reconnectError.wasSharing);
+          }
+          throw new Error('Cannot re connect the meeting , please rejoin.');
```

Comment:
Do you want to keep the original `reconnectError` object as well?

Reply:
@mwegman  the parent function already throws that error to the developer

Reply:
```suggestion
          throw new AggregateError([reconnectError], 'Cannot re connect the meeting , please rejoin.');
```
You could use the AggregateError

Reply:
@mwegman i did remove that as the needTOrejoin error should be sufficient